agent/main.go

```diff
@@ -280,3 +288,7 @@ func getConfig(localURL *url.URL) (*AgentConfig, error) {
 		Certificate: tlsCert,
 	}, err
 }
+
+func setSysctl() error {
+	return ioutil.WriteFile(path.Join(base, sysctlBridgeCallIPTables), []byte(strconv.Itoa(1)), 0640)
```

Comment:
Can you errors.Wrap this to set a message indicating you failed to write 1 to /proc.....  Otherwise the user gets a random "I/O error" with no idea whats up.